timestamp-impl/src/main/java/com/palantir/timestamp/DebugLogger.java

```diff
@@ -28,9 +28,28 @@
  */
 @SuppressFBWarnings("SLF4J_LOGGER_SHOULD_BE_PRIVATE")
 public final class DebugLogger {
-    public static final Logger logger = LoggerFactory.getLogger(DebugLogger.class);
+    private static final Logger logger = LoggerFactory.getLogger(DebugLogger.class);
 
     private DebugLogger() {
         // Logging utility class
     }
+
+    public static void handedOutTimestamps(TimestampRange range) {
```

Comment:
This was a shared static `Logger` object before, so we never had information about which class it was coming from. 

According to the docs on `DebugLogger` it's specifically meant to trace events that could be helpful in debugging `MultipleRunningTimestampService` errors - so I believe the goal was to allow all the logs going through this class can be enabled or disabled together in config.